index.html

```diff
@@ -345,6 +345,46 @@ <h2>About Myself</h2>
             <hr />
           </li>
         </ul>
+        <form
+          action="https://formspree.io/f/xwkjylde"
+          method="POST"
+          class="form-section"
+        >
+          <label class="title-form">Contact me</label>
+          <p class="subtitle-form">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it!
+          </p>
+          <ul class="form-ul">
+            <li>
+              <input
+                type="text"
+                maxlength="30"
+                name="user_name"
+                placeholder="Name Lastname"
+                required
+              />
+            </li>
+            <li>
+              <input
+                type="email"
+                name="user_email"
+                placeholder="loremipsum@gmail.com"
+                required
+              />
+            </li>
+            <li>
+              <textarea
+                type="message"
+                name="user_message"
+                placeholder="Write your message here..."
+                maxlength="550"
+              ></textarea>
```

Comment:
[Optional] When new text is added, the font color shows it in the same way as the placeholder, which looks as if the text field is disabled and could confuse users, consider using the same font color as the other input fields so the contrast makes clear that the field is not disabled